packages/neos-ui-backend-connector/package.json

```diff
@@ -20,7 +20,8 @@
   },
   "dependencies": {
     "@neos-project/neos-ui-extensibility": "8.0.0",
-    "@neos-project/utils-redux": "8.0.0"
+    "@neos-project/utils-redux": "8.0.0",
+    "@neos-project/neos-ts-interfaces": "8.0.0"
```

Comment:
TS2307: Cannot find module '@neos-project/neos-ts-interfaces'

so we need to add it as dependency to be present in the node_modules folder of this sub package